thirdeye/thirdeye-frontend/app/pods/rootcause/controller.js

```diff
@@ -488,6 +490,70 @@ export default Ember.Controller.extend({
     return aggregates;
   },
 
+  /**
+   * Returns object that represents mapping between attributes in events and input values in config
+   * Example:
+   * {
+   *  holiday: {
+   *    countryCode: ['US', 'IN', 'MX'],
+   *    region: ['North America', 'Antarctica']
+   *  },
+   *  deployment: {
+   *    fabric: ['prod', 'local']
+   *  }
+   * }
+   * @type {Object}
+   */
+  attributesMap: Ember.computed(
```

Comment:
this should be encapsulated within filter-bar. entities go in, a set of urns comes out via onSelection action - i.e. "entities" and "onSelection" is the interface